src/components/header/renderer.tsx

```diff
@@ -157,14 +161,27 @@ class Header extends React.Component<Props, State> {
     this.setState({
       invalidUrl: !gistCommits.ok,
     });
+    if (this.state.invalidUrl) {
```

Comment:
Why do we condition on having an invalid URL?

Reply:
If the URL is invalid and length of revision/filename is non zero, should I show the revision/filename error?

Reply:
I don't know. What's the goal of this logic?